experimental/addons/gitops/definitions/gitops.cue

```diff
@@ -0,0 +1,46 @@
+gitops: {
+	annotations: {}
+	attributes: workload: definition: {
+		apiVersion: "apps/v1"
+		kind: "Deployment"
+	}
+	description: "KubeVela addon for implementing GitOps for continuous deployment using either fluxcd or argocd"
+	type: "component"
+	labels: {}
+}
+
+template: {
+	output: {
+		if parameter.agent == "fluxcd" {
```

Comment:
I suggest that you can separate the entire CR of fluxcd and argocd with if, not just the apiVersion.

Reply:
Sure, I am working on that now.